staging/src/k8s.io/legacy-cloud-providers/aws/aws.go

```diff
@@ -3451,22 +3446,11 @@ func getPortSets(annotation string) (ports *portSets) {
 // Extra groups can be specified via annotation, as can extra tags for any
 // new groups. The annotation "ServiceAnnotationLoadBalancerSecurityGroups" allows for
 // setting the security groups specified.
-func (c *Cloud) buildELBSecurityGroupList(serviceName types.NamespacedName, loadBalancerName string, annotations map[string]string) ([]string, error) {
+func (c *Cloud) buildELBSecurityGroupList(serviceName types.NamespacedName, loadBalancerName string, annotations map[string]string) ([]string, bool, error) {
```

Comment:
I think we want to return a map[string]bool to make it possible to express that only the kubernetes-created securitygroup should be setup. I don't mind complicating this function's signature as long as it's documented and tested

Reply:
So this is ok I understad as it is exactly doing this (the bool is only true if a SG has been setup because neighter `service.beta.kubernetes.io/aws-load-balancer-security-groups` was specified nor a global `ElbSecurityGroup`.